interfaces/portal/src/app/utils/csv-helpers.ts

```diff
@@ -1,7 +1,23 @@
 import Papa from 'papaparse';
 
 export const unknownArrayToCsvBlob = (data: unknown[]): Blob => {
-  const csvContent = Papa.unparse(data);
+  // We need to account for not all objects having the same attributes.
+  const columns = new Set<string>();
+  if (typeof data === 'object' && 'data' in data && Array.isArray(data.data)) {
+    data.data.forEach((item) => {
```

Comment:
I don't understand how this is `data.data` because according to the definition of the argument, `data` itself should be an array 🤔